src/components/molecules/GeoJsonForm.vue

```diff
@@ -0,0 +1,160 @@
+<script lang="ts">
+import { defineComponent, ref } from 'vue';
+import {
+  drawnShape, specifiedShape, geometryInputSelection,
+} from '@/store/search';
+import { useMap } from '@/store/search';
+
+export default defineComponent({
+  name: 'GeoJsonForm',
+
+  setup() {
+
+    const geoOptions = [
+      'Draw on Map',
+      'Copy/Paste GeoJson',
+      'Upload File',
+    ];
+    const geoJsonString = ref('');
+    const geoJsonErrorMessages = ref(['']);
+
+    const clearShape = () => {
+      drawnShape.value = {
+        type: 'Polygon',
+        coordinates: [],
+      };
+      geoJsonErrorMessages.value = [];
+    };
+    const selectShape = (value: Event) => {
+      clearShape();
+      geometryInputSelection.value = value;
+    };
+
+    const isGeoJSON = (inputText: string) => {
+      try {
+        JSON.parse(inputText);
+        geoJsonErrorMessages.value = [];
+      } catch (e) {
+        geoJsonErrorMessages.value = ['Not a valid GeoJSON Polygon or MultiPolygon'];
+      }
+      return true;
+    };
```

Comment:
This might just be a personal preference on my part, but when I see a function named like `isXXX`, I expect it to be a pure function that simply checks if something is true. This function does that, but also has the side effect of populating the `geoJsonErrorMessages` list.

Also, the return type should probably be `boolean` in all cases -
```suggestion
    const isGeoJSON = (inputText: string) => {
      try {
        JSON.parse(inputText);
        geoJsonErrorMessages.value = [];
      } catch (e) {
        geoJsonErrorMessages.value = ['Not a valid GeoJSON Polygon or MultiPolygon'];
        return false;
      }
      return true;
    };
```